poucave/html/js/app.js

```diff
@@ -246,12 +249,23 @@ class Overview extends Component {
           <div class="card-body text-center">
             <i class="fa fa-4x ${iconClass}"></i>
             <p>
-              <strong>The current system status is ${isHealthy ? "Healthy" : "Unhealthy"}.</strong>
+              <strong>The system ${isHealthy ? "is currently healthy" : "has failing checks"}.</strong>
               <br />
               <span class="text-gray-medium">
                 Last updated <${TimeAgo} date="${new Date()}" />.
               </span>
             </p>
+            <p>
+              <${FocusedCheck.Consumer}>
+                ${focusedCheckContext => (
+                  failing.map(r => (
+                    html`<li>
+                      <a href="#" onClick=${() => focusedCheckContext.setValue(r.project, r.name)}>${r.project} / ${r.name}</a>
+                    </li>`
+                  ))
+                )}
+              </>
+            </p>
```

Comment:
also it might be better to move this whole thing into a new method like:
```js
renderErrorList() {
  const failing = Object.values(results).filter(r => !r.isLoading && !r.success);
  if (failing.length > 0) {
    return "";
  }

  return html`
    <ul>
      ...
    </ul>
  `;
}
```

and then use here as:
```js
${this.renderErrorList()}
```

this will prevent an empty `<ul>`/`<p>` from being output.